src/remote/watcher.coffee

```diff
@@ -77,8 +79,7 @@ class RemoteWatcher
                         @whenReady callback
             else
                 @changes = @couch.client.changes
-                    filter: (doc) ->
-                        doc.docType?.toLowerCase() in ['file', 'folder']
+                    filter: @filterName
```

Comment:
why ? I think is more complicate. It's a filter so the function is on filter field I don't like this way to do this.